phalcon/mvc/model/query.zep

```diff
@@ -2513,6 +2522,93 @@ class Query implements QueryInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * gets the read connection from the model if there no transaction inside the query object
```

Comment:
cool :) thx ! :) and the code examples will be added to model::find and criteria::execute.